src/tile.js

```diff
@@ -805,17 +844,22 @@ $.Tile.prototype = {
     },
 
     /**
-     * Removes tile from its container.
-     * @function
+     * Removes tile from the system: it will still be present in the
+     * OSD memory, but marked as loaded=false, and its data will be erased.
+     * @param {boolean} [erase=false]
      */
-    unload: function() {
-        //TODO AIOSA remove this.element and move it to a data constructor
-        if ( this.imgElement && this.imgElement.parentNode ) {
-            this.imgElement.parentNode.removeChild( this.imgElement );
-        }
-        if ( this.element && this.element.parentNode ) {
-            this.element.parentNode.removeChild( this.element );
+    unload: function(erase = false) {
```

Comment:
In the tests, I realized I might want to call unload manually (for whatever reasons), and so I allowed it to be called so that data/references is not damaged.

Reply:
I am not sure. You can ask for a tile to be deleted.... I mean it could be useful...

Reply:
Cool... just wanting to be thoughtful about what we add to the API.